doc/version_history.rst

```diff
@@ -6,6 +6,11 @@
 Version History
 ===============
 
+v1.19.3
```

Comment:
I think adding a new script should probably be a new minor release.. 

we have several PRs open so we should make this batch `v1.20.0`. So, once you merge this PR we should wait.